app/src/androidTest/java/com/arygm/quickfix/ui/authentication/InfoScreenTest.kt

```diff
@@ -121,6 +121,9 @@ class InfoScreenTest {
 
     // Click the button and verify navigation
     composeTestRule.onNodeWithTag("nextButton").performClick()
+
+    composeTestRule.mainClock.advanceTimeBy(500L)
+
```

Comment:
You can maybe use the wait function with the condition of the navigation action invocations being empty, but with a reasonable waiting time.